packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -113,44 +113,64 @@ export default class MuiDetailGenerator implements DetailGenerator {
               if (ts.isJsxAttribute(prop)) {
                 const propName = prop.name.escapedText.toString();
                 const inputProp = widgetProperties.properties.find(l => l.name === propName);
-
                 if (inputProp) {
                   if (prop.initializer) {
-                    if (ts.isStringLiteral(prop.initializer)) {
-                      if(inputProp.value !== prop.initializer.text) {
+                    const initializer = prop.initializer
+                    if (inputProp.type === "STRING_LITERAL") {
```

Comment:
let's have a helper function for each branch
STRING_LITERAL
EXPRESSION
OTHERVISE

naming of this function will help understand this code (proper naming helps understand what is the intent) ...